src/main/java/com/rackspace/ceres/app/validation/StringValidator.java

```diff
@@ -17,18 +17,21 @@
 
 package com.rackspace.ceres.app.validation;
 
+import java.util.Arrays;
 import org.springframework.stereotype.Component;
 import org.springframework.util.StringUtils;
 
 @Component
-public class MetricNameAndGroupValidator {
+public class StringValidator {
 
-  public void validateMetricNameAndMetricGroup(String metricName, String metricGroup) {
-    if(!StringUtils.hasText(metricGroup) && !StringUtils.hasText(metricName)) {
+  public static void validateRequest(String[] strArr) {
```

Comment:
instead of taking string array can we make it var args of string type?

Reply:
and also method can be changed to something meaningful or you can add comment above method because right now it doesn't give much hint what this method is validating

Reply:
can change exception message too as it will not always be metric group and metric name since you made it generic

Reply:
but then how to propagate a meaningful message after violation of different metricName/metricGroup contraints ?